package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "react-icons": "^4.4.0"
```

Comment:
@Nyle-FBM Does this have to be here or can it be moved to /client/package.json?

Reply:
tbh idek. but all that's for is the pdf symbol which i'll probably change anyway

Reply:
yes

Reply:
okay no worries lol i was just tryna be nice. it actually should be in the client folders package json. what you likely did was an "npm install react-icons" inside "patient-management-system" folder but it needed to be done in the client folder. All you gotta do is delete the package json and package lock json files and do the "npm install react-icons" inside client

Reply:
* delete the json files from the "patient-management-system" folder

Reply:
i mean